### ISSUE TITLE
This bundle is disabling symfony some error pages.

### ISSUE THREAD

**Issue by stacyharper** [IDCI-Consulting/DocumentManagementBundle#5]
Hello,
I lost time debuging my app, trying to find why there is not error page anymore.
Jumping in commits like Rick and Morty, I found that this error started far away when I started to use DocumentManagementBundle.
I then tried to check the bundle config.yml and I found that this lines where in cause !

```yaml
fos_rest:    
    exception:
        codes: ~
        messages: ~
```

I remember you uses this to solve some problems but I think we should try to find another solutions !

I'll try to find an alternative that will not break my app :D
I'll let you know my victories and I'll make a MR.

**Comment by stacyharper** [comment]
It seems that tests pass without this config. Why did you use this ?

**Comment by burahimu** [comment]
Fixed with #6
